Template variables error after upgrade to 5.3.0

### What Grafana version are you using?
5.3.0
### What datasource are you using?
Postgresql
### What OS are you running grafana on?
Ubuntu 18.04
### What did you do?
Upgrade from 5.2.4, using working dashboard (on 5.2.4)
### What was the expected result?
Smoothly upgrade
### What happened instead?
Got error as shown in the second screenshots

The dashboard uses 2 template variables:
The first one seems to work properly.
![screenshot 1](https://user-images.githubusercontent.com/44204889/47027279-7637b880-d167-11e8-9e9f-92fddbb6b12a.png)

The second variable depends from the previous one and, when I try to change any property, I got an error:
![screenshot 2](https://user-images.githubusercontent.com/44204889/47027456-c9117000-d167-11e8-92ad-66254031cebc.png)
I got similar error when I try to open the dashboard. The second variable in the panel does not contain any values.

It seems to me that there is no second template variable query, I see the first one only.

Chrome console shows:
```
warning rootScope onAppEvent called without localscope
grafana_app.ts:86 warning rootScope onAppEvent called without localscope
history_srv.ts:33 Templating init failed TypeError: Cannot read property 'queryModel' of undefined
    at datasource.ts:108
    at Qt (lodash.js:639)
    at Function.Ka [as map] (lodash.js:9553)
    at e.interpolateVariable (datasource.ts:106)
    at e.formatValue (template_srv.ts:119)
    at template_srv.ts:247
    at String.replace (<anonymous>)
    at e.replace (template_srv.ts:218)
    at e.metricFindQuery (meta_query.ts:22)
    at e.metricFindQuery (query_variable.ts:172)

```
This is the firefox console output:
```
commitChanges	http://localhost:3000/public/build/app.09796ed34e70bee7294e.js:1:67617
l 		http://localhost:3000/public/build/app.09796ed34e70bee7294e.js:1:68545
eval 		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:99313
apply 		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:99537
l 		http://localhost:3000/public/build/app.09796ed34e70bee7294e.js:1:68525
dispatch 	http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:124:37882
handle 		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:124:35947

Templating init failed TypeError: "a is undefined"
interpolateVariable	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:260504
Qt		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:9:5768
Ka		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:9:40614
interpolateVariable	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:260482
formatValue	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:52532
replace		http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:54032
replace		http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:53764
metricFindQuery	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:261835
metricFindQuery	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:65725
updateOptionsFromMetricFindQuery	http://localhost:3000/public/build/5.09796ed34e70bee7294e.js:10:65383
l		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:92198
l		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:92043
digest		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:97575
apply		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:99590
r		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:99735
h		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:33036
n		http://localhost:3000/public/build/vendor.09796ed34e70bee7294e.js:130:34396  

```

Thanks,
Pino

--- a/public/app/plugins/datasource/postgres/datasource.ts
+++ b/public/app/plugins/datasource/postgres/datasource.ts
@@ -50,7 +50,7 @@ export class PostgresDatasource {
         intervalMs: options.intervalMs,
         maxDataPoints: options.maxDataPoints,
         datasourceId: this.id,
-        rawSql: queryModel.render(this.interpolateVariable),
+        rawSql: queryModel.render((value, variable) => this.interpolateVariable(value, variable)),
         format: target.format,
       };
     });
@@ -82,7 +82,9 @@ export class PostgresDatasource {
     const query = {
       refId: options.annotation.name,
       datasourceId: this.id,
-      rawSql: this.templateSrv.replace(options.annotation.rawQuery, options.scopedVars, this.interpolateVariable),
+      rawSql: this.templateSrv.replace(options.annotation.rawQuery, options.scopedVars, (value, variable) =>
+        this.interpolateVariable(value, variable)
+      ),
       format: 'table',
     };
 
@@ -108,7 +110,7 @@ export class PostgresDatasource {
     const interpolatedQuery = {
       refId: refId,
       datasourceId: this.id,
-      rawSql: this.templateSrv.replace(query, {}, this.interpolateVariable),
+      rawSql: this.templateSrv.replace(query, {}, (value, variable) => this.interpolateVariable(value, variable)),
       format: 'table',
     };
 

--- a/public/app/plugins/datasource/postgres/datasource.ts
+++ b/public/app/plugins/datasource/postgres/datasource.ts
@@ -20,7 +20,7 @@ export class PostgresDatasource {
     this.interval = (instanceSettings.jsonData || {}).timeInterval;
   }
 
-  interpolateVariable(value, variable) {
+  interpolateVariable = (value, variable) => {
     if (typeof value === 'string') {
       if (variable.multi || variable.includeAll) {
         return this.queryModel.quoteLiteral(value);
@@ -37,7 +37,7 @@ export class PostgresDatasource {
       return this.queryModel.quoteLiteral(v);
     });
     return quotedValues.join(',');
-  }
+  };
 
   query(options) {
     const queries = _.filter(options.targets, target => {
@@ -50,7 +50,7 @@ export class PostgresDatasource {
         intervalMs: options.intervalMs,
         maxDataPoints: options.maxDataPoints,
         datasourceId: this.id,
-        rawSql: queryModel.render((value, variable) => this.interpolateVariable(value, variable)),
+        rawSql: queryModel.render(this.interpolateVariable),
         format: target.format,
       };
     });
@@ -82,9 +82,7 @@ export class PostgresDatasource {
     const query = {
       refId: options.annotation.name,
       datasourceId: this.id,
-      rawSql: this.templateSrv.replace(options.annotation.rawQuery, options.scopedVars, (value, variable) =>
-        this.interpolateVariable(value, variable)
-      ),
+      rawSql: this.templateSrv.replace(options.annotation.rawQuery, options.scopedVars, this.interpolateVariable),
       format: 'table',
     };
 
@@ -110,7 +108,7 @@ export class PostgresDatasource {
     const interpolatedQuery = {
       refId: refId,
       datasourceId: this.id,
-      rawSql: this.templateSrv.replace(query, {}, (value, variable) => this.interpolateVariable(value, variable)),
+      rawSql: this.templateSrv.replace(query, {}, this.interpolateVariable),
       format: 'table',
     };
 

